qlty-cli/src/initializer/scanner.rs

```diff
@@ -353,11 +353,13 @@ impl Scanner {
         }
     }
 
-    fn compute_default_ignores(&mut self) {
-        self.default_ignores = self.default_config.ignore.clone();
-        self.default_ignores
+    // Not sure why we have this?
+    // Maybe it should be removed.
```

Comment:
We should address this comment before merging.

Reply:
Seems right, I think we can safely remove it but we should probably do that in a tiny, isolated change later